appinventor/blocklyeditor/src/blocks/components.js

```diff
@@ -170,6 +170,13 @@ Blockly.Blocks.component_event = {
     if (this.instanceName == oldname) {
       this.instanceName = newname;
       this.componentDropDown.setValue(this.instanceName);
+      if (this.isCollapsed())
+      {
+        var COLLAPSED_INPUT_NAME = '_TEMP_COLLAPSED_INPUT';     
```

Comment:
If it is used as a constant, then that is fine (sorry if I missed that when
I first looked at the code).

-Jeff
On Jun 6, 2015 1:56 PM, "Shruti Rijhwani" <notifications@github.com> wrote:

> In appinventor/blocklyeditor/src/blocks/components.js
> <https://github.com/mit-cml/appinventor-sources/pull/409#discussion_r31870099>
> :
>
> > @@ -170,6 +170,13 @@ Blockly.Blocks.component_event = {
> >      if (this.instanceName == oldname) {
> >        this.instanceName = newname;
> >        this.componentDropDown.setValue(this.instanceName);
> > +      if (this.isCollapsed())
> > +      {
> > +        var COLLAPSED_INPUT_NAME = '_TEMP_COLLAPSED_INPUT';
>
> @jisqyv <https://github.com/jisqyv>, what variable name should be
> changed?
>
> COLLAPSED_INPUT_NAME in this case is used as a constant string.
>
> Is there a better way to write this code?
>
> Thanks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-cml/appinventor-sources/pull/409/files#r31870099>.
>